components/monitoring/logging/staging/external-secrets/kustomization.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-- monitoring-logging-splunk-log-forwarder.yaml
```

Comment:
Not seeing deletion for this (I suppose now no longer needed) file in the PR.

Reply:
@ifireball I'm a bit confused. It was deleted and I also see it deleted in the PR.

Reply:
ok, now I see it. NVM.